hugo/assets/scss/components/teaser.scss

```diff
@@ -29,45 +27,129 @@
 
         color: $c-blue--dark;
         font-size: 1.5rem;
-        margin: 0 0 0.4rem;
+        margin: 0 0 .4rem;
     }
 
     &__meta {
         color: var(--teaser-meta-color, $c-grey--dark);
-        font-size: 0.8125rem;
+        font-size: .8125rem;
         line-height: 1.3;
-        margin: 0 0 0.4rem;
+        margin: 0 0 .4rem;
     }
 
     &__excerpt {
         font-weight: $weight-medium;
-        margin: 0.5rem 0 0;
+        margin: .5rem 0 0;
     }
 
     &__link {
         @include stretch;
 
         border: 2px solid transparent;
         border-radius: $b-radius;
-        transition: border-color 0.2s;
+        transition: border-color .2s, background-color .2s;
 
         span {
             @include sr-only;
         }
 
         &:focus,
         &:hover {
-            background: transparentize($c-blue, 0.94);
+            background: transparentize($c-blue, .94);
+            transition: background-color .2s;
```

Comment:
putting the transition on focus itself means it only transitions one way around, so when focus > no focus and not the other way around anymore